pawno/include/PPC_Common.inc

```diff
@@ -241,14 +241,20 @@ CommandList_Create(playerid)
 }
 
 // This function returns "1" if the given vehicle-id is a plane or helicopter
-IsVehicleAirVehicle(vid)
+IsVehicleAirModel(vid)
 {
 	switch (GetVehicleModel(vid))
 	{
 		case 592, 577, 511, 512, 593, 520, 553, 476, 519, 460, 513, 548, 425, 417, 487, 488, 497, 563, 447, 469: return 1;
-		default: return 0;
 	}
-
+	return 0;
```

Comment:
You can remove that as  `default` already returns zero if no  `case` has been called.